docs/spark/spark-writes.md

```diff
@@ -287,13 +287,14 @@ To run a CTAS or RTAS, use `create`, `replace`, or `createOrReplace` operations:
 
 ```scala
 val data: DataFrame = ...
-data.writeTo("prod.db.table").create()
+data.writeTo("prod.db.table").using("iceberg").create()
```

Comment:
This depends on the catalog. If you're using an Iceberg catalog, it shouldn't be necessary to do this.

Reply:
Thanks for pointing that out. I did not actually know that. I usually use the `SparkSessionCatalog`.

Reply:
For the `SparkSessionCatalog`, you do need this. It might be better just to recommend it here, although it isn't needed unless we are trying to determine whether to pass the create through to the underlying catalog.

Reply:
I reverted the change and added a clarification for when you use `SparkSessionCatalog`. Please let me know if you think it is better now.